daprdocs/content/en/operations/hosting/kubernetes/cluster/setup-eks.md

```diff
@@ -49,11 +87,37 @@ This guide walks you through installing an Elastic Kubernetes Service (EKS) clus
    --source-group [your_security_group]
    ```
 
+2. Dapr 1.14 Scheduler service requires a default storage class to be set. If you don't have one, patch your cluster with the command below: 
```

Comment:
Suggest to change `Dapr 1.14 Scheduler service requires a default storage class to be set` to 'Add a default storage class if you don't have one:`.

Reply:
Done